### ISSUE TITLE
Microsoft Edge

### ISSUE THREAD

**Issue by NRGSystems-RickDoll** [SteveSandersonMS/BlazorInputFile#1]
Can't seem to drag and drop files using the Edge browser onto the input file box.

**Comment by pos123** [comment]
Yes same here. I would like to see this working. We use Microsoft Edge at work and don't really have the choice to use anything else.

**Comment by ankitavyas** [comment]
Did anyone manged to get it work ?

**Comment by ankitavyas** [comment]
> Yes same here. I would like to see this working. We use Microsoft Edge at work and don't really have the choice to use anything else.

Have you had a solution?

**Comment by ankitavyas** [comment]
> Can't seem to drag and drop files using the Edge browser onto the input file box.

Have you had a solution?

**Comment by DataJuggler** [comment]
In all my years I have never dragged a file for file upload. I use Edge every day for file uploads.

I have always browsed for a file to upload.
